documentation/docs/20-core-concepts/40-page-options.md

```diff
@@ -76,11 +76,12 @@ For _pages_, we skirt around this problem by writing `foo/index.html` instead of
 
 ### Troubleshooting
 
-If you encounter an error like 'The following routes were marked as prerenderable, but were not prerendered' it's because the route in question (or a parent layout, if it's a page) has `export const prerender = true` but the page wasn't actually prerendered, because it wasn't reached by the prerendering crawler.
+If you encounter an error like 'The following routes were marked as prerenderable, but were not prerendered' it's because the route in question (or a parent layout, if it's a page) has `export const prerender = true` but the page wasn't reached by the prerendering crawler and thus wasn't prerendered.
 
-Since these routes cannot be dynamically server-rendered, this will cause errors when people try to access the route in question. There are two ways to fix it:
+Since these routes cannot be dynamically server-rendered, this will cause errors when people try to access the route in question. There are three ways to fix it:
```

Comment:
```suggestion
Since these routes cannot be dynamically server-rendered, this will cause errors when people try to access the route in question. There are a few ways to fix it:
```